### PR TITLE
Mobile code challenge proposal

### PR DESCRIPTION
**Pull request by albertmiro**
Android / iOS tech task

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,248 @@
-# Mobile-coding-challenge
-Android/iOS tech task
+# Yoga Solo -  Android/iOS Code challenge
+
+Congratulations! You’re nearly there to join our team as Android/iOS developer! 
+
+We have a really big issue in our office and we need you to solve it. We are trying to practice Yoga, but we are not able to remember the different postures, and we need your superpowers as Android developer to provide us a solution for it!
```
Comment (WildStudio) [inline README.md]:
```suggestion
We have a really big issue in our office and we need you to solve it. We are trying to practice Yoga, but we are not able to remember the different postures, and we need your superpowers as Android/iOS developer to provide us a solution for it! 
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,248 @@
...
+*Example mock-up for the dynamic height:*
+
+<image src="./images/mockup_list.jpg"></image>
+
+## What you should do?
+
+You should create an Android app in which we would be able to see the postures (retrieved from a server) and the detail of each posture.
+To retrieve data from a server, you should create your API with the data we provide you.
+
+Don't worry! This is not a Full Stack code challenge ;-) you can use services like https://github.com/typicode/json-server to get quickly a server running from where you can start retrieving data.
```
Comment (WildStudio) [inline README.md]:
```suggestion
Don't worry! This is not a Full Stack code challenge ;-) you can use services like https://github.com/typicode/json-server to get a server up running quickly. From there you can start retrieving the data.
```

Comment (albertmiro) [reply README.md]: Nice one, thanks!

File: README.md (hunk 1 above)
Comment (albertmiro) [reply README.md]: Ups! 😂

**Pull request merged**